go-testing-tutorial/main.go

```diff
@@ -54,7 +54,7 @@ func (api *API) searchHandler(w http.ResponseWriter, r *http.Request) {
 		}
 
 		searchResults := SearchResults{
-			Results: []string{"Cutie", "Autism", "iPhone 12"},
+			Results: api.repository.search("whatever"),
```

Comment:
This shouldn't be hardcoded to "whatever', it should be whatever the query string was.